doc/sailfish-webview-webviewadditional.qdoc

```diff
@@ -0,0 +1,321 @@
+web view, but in some cases clients may wish to set the viewport width
+explicitly (e.g. to show some chrome at one edge or the other, without
+obscuring some content underneath, while keeping the geometry of the web view
+item itself unchanged).
+
+Device orientation should be considered when binding to this property.
+
+See also: \l{WebView::contentRect}, \l{WebView::orientation}
+
+\section2 WebView::viewportHeight
+
+\c{qreal}-type property.
+
+The height of the webview viewport, in device pixels, which by default is
+bound to the screen height (in portrait orientation) or width (in landscape
+orientation).
+
+Note that the viewport height will usually be the same as the height of the
+web view, but in some cases clients may wish to set the viewport height
+explicitly (e.g. to show some chrome at one edge or the other, without
+obscuring some content underneath, while keeping the geometry of the web view
+item itself unchanged).
+
+Device orientation should be considered when binding to this property.
+
+See also: \l{WebView::contentRect}, \l{WebView::orientation}
+
+\section2 WebView::atXBeginning
+
+\c{bool}-type read-only property.
+
+Whether the viewport of the webview is scrolled to the far-left of the content.
+
+Effectively, the value of this property will be true if the
+\l{WebView::scrollableOffset} x-value is approximately zero.
+
+See also: \l{WebView::atXEnd}, \l{WebView::scrollableOffset}
+
+\section2 WebView::atXEnd
+
+\c{bool}-type read-only property.
+
+Whether the viewport of the webview is scrolled to the far-right of the
+content.
+
+Effectively, the value of this property will be true if the
+\l{WebView::scrollableOffset} x-value plus the (\l{WebView::resolution} scaled)
+\l{WebView::contentRect} width is approximately equal to or greater than the
+\l{WebView::scrollableSize} width.
+
+See also: \l{WebView::atXBeginning}, \l{WebView::scrollableOffset},
+\l{WebView::scrollableSize}, \l{WebView::contentRect}
+
+\section2 WebView::atYBeginning
+
+\c{bool}-type read-only property.
+
+Whether the viewport of the webview is scrolled to the top of the content.
+
+Effectively, the value of this property will be true if the
+\l{WebView::scrollableOffset} y-value is approximately zero.
+
+See also: \l{WebView::atYEnd}, \l{WebView::scrollableOffset}
+
+\section2 WebView::atYEnd
+
+\c{bool}-type read-only property.
+
+Whether the viewport of the webview is scrolled to the bottom of the
+content.
+
+Effectively, the value of this property will be true if the
+\l{WebView::scrollableOffset} y-value plus the (\l{WebView::resolution} scaled)
+\l{WebView::contentRect} height is approximately equal to or greater than the
+\l{WebView::scrollableSize} height.
+
+See also: \l{WebView::atYBeginning}, \l{WebView::scrollableOffset},
+\l{WebView::scrollableSize}, \l{WebView::contentRect}
+
+\section2 WebView::dragging
+
+\c{bool}-type read-only property.
+
+Whether the user is currently dragging the scrollable area.
```

Comment:
It might be worth mentioning that this won't be true if content within the page is being dragged (i.e. the touch events are being consumed by the page).

Reply:
Very good point, done